params/superchain.go

```diff
@@ -83,10 +79,6 @@ func LoadOPStackChainConfig(chainID uint64) (*ChainConfig, error) {
 		},
 	}
 
-	// note: no actual parameters are being loaded, yet.
-	// Future superchain upgrades are loaded from the superchain chConfig and applied to the geth ChainConfig here.
-	_ = superchainConfig.Config
-
 	// special overrides for OP-Stack chains with pre-Regolith upgrade history
 	switch chainID {
```

Comment:
all these should be transferred to the SCR PR

Reply:
I just did the `RegolithTimes` so far, to prove out the approach. Block times aren't yet in the registry at all. But we can discuss that as a follow up step.

Reply:
Agree, let's only transfer fork times for now.